browser-test/test.endtoend.ts

```diff
@@ -0,0 +1,546 @@
+import * as assert from 'assert';
```

Comment:
as discussed - can we split this huge test file into two?

Reply:
When you split it, for the code that was generated and copied here, please put a header comment at the top of them file that states something to the effect of "This code was generated by XXX. DO NOT EDIT." 

This way, I know not to send PRs to change it.